src/matchers/toBeAccessible/index.spec.ts

```diff
@@ -109,4 +109,21 @@ test.describe.parallel('toBeAccessible', () => {
     expect(duration).toBeGreaterThan(1000)
     expect(duration).toBeLessThan(1500)
   })
+
+  test('should allow providing custom report filename', async ({ page }) => {
+    test.fail()
+    const reportFileName = 'custom-report.html'
+    const content = await readFile('inaccessible.html')
+    await page.setContent(content)
+    await expect(page).toBeAccessible({ timeout: 2000, reportFileName })
+    await expect(reportFileExists(test.info(), reportFileName)).toBe(true)
+  });
+
+  test(`should use default report filename when a custom one is not provided`, async ({ page }) => {
```

Comment:
Can we move this above the other test?  We prefer to put default tests before overrides.

```suggestion
  test('default report filename', async ({ page }) => {
```